### PR TITLE
Use gcr.io instead of docker.io

### PR DESCRIPTION
**Pull request by kflynn**
...since we're seeing problems from at least some EKS users pulling from docker.io.

### CODE REVIEW

File: ci/02-oss-values.yaml [hunk 1]
```diff
@@ -1,7 +1,7 @@
 # install the Ambassador API Gateway
 image:
-  repository: docker.io/datawire/ambassador
-  tag: 1.5.5
```
Comment (cakuros) [inline ci/02-oss-values.yaml:4]: wtf, how long has this tag been at 1.5.5?

**Review by cakuros — APPROVED** [review]
LGTM

File: ci/02-oss-values.yaml (hunk 1 above)
Comment (LukeShu) [inline ci/02-oss-values.yaml:4]: O.o

**Review by LukeShu — APPROVED** [review]
lgtm

File: ci/02-oss-values.yaml (hunk 1 above)
Comment (kflynn) [reply ci/02-oss-values.yaml:4]: No idea. Pretty sure that the test is just "can we install from Helm", but it should be updated regularly.

**Comment by kflynn** [comment]
This was for Game Day. Whew. 😉

**Pull request closed without merging**